backgroundremover/u2net/detect.py

```diff
@@ -3,13 +3,23 @@
 import sys
 import numpy as np
 import torch
+import logging
 from hsh.library.hash import Hasher
 from PIL import Image
 from torchvision import transforms
 
 from . import data_loader, u2net
 from .. import github
 
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+logger.propagate = False
+
+if not logger.handlers:
+    handler = logging.StreamHandler()
+    handler.setFormatter(logging.Formatter('%(levelname)s - %(message)s'))
+    logger.addHandler(handler)
```

Comment:
You should not set settings like level or add handlers here. Handlers should only be added by end-users (other libraries or the main entry point).

```suggestion
```